src/components/item/sharing/ItemMembershipSelect.tsx

```diff
@@ -71,9 +56,9 @@ const ItemMembershipSelect = ({
       values={Object.values(PermissionLevel).map((v) => ({
         value: v,
         text: enumT(v),
-        disabled: disabledMap[v],
+        disabled,
```

Comment:
I don't like that we have to disable all options for the select to be disabled. This is not clean.